src/listeners/interactionCreate.ts

```diff
@@ -136,8 +137,16 @@ client.on("interactionCreate", async interaction => {
 								console.error(`Couldn't get string ${path} in English for ${file}, please fix this`)
 						}
 					}
-					if (typeof string === "string" && variables)
+					if (typeof string === "string" && variables) {
 						for (const [variable, text] of Object.entries(variables)) string = string.replace(`%%${variable}%%`, String(text))
+						const discordLocale = transformBotLocale(lang)!
```

Comment:
I'm gonna repeat the question: why does this only support the locales Discord supports and not all the ones we support on the bot?

Reply:
The library supports only `xx-XX` format locales

Reply:
Ok? Then transform them

Reply:
> Ok? Then transform them

The bot locales format is `xx_XX`, the library supports `xx-XX`.